docs/DeveloperGuide.md

```diff
@@ -387,11 +387,29 @@ otherwise)
 
 **MSS**
 
-1. User wants to look for items that are expiring soon.
+1. User search for items that are expiring soon.
 2. StoreMando returns a list of expiring items.
 
    Use case ends.
 
+**Extensions**
+
+* 1a. User input number is a negative number.
```

Comment:
I think this should be written as "User inputs a negative number".

Reply:
All right thanks!